config/fypp_deployment.py

```diff
@@ -0,0 +1,165 @@
+import os
+import fypp
+import argparse
+from joblib import Parallel, delayed
```

Comment:
I am just trying the script now on a different machine. It could be good to mention somewhere the these should be installed first (if it is not already the case, of course).

Reply:
Oh yes, that was the main purpose of the requirements.txt file, to simply do `pip install -r requirements.txt`. I'll add those and the comment in the README.

Reply:
Done on commit https://github.com/fortran-lang/stdlib/pull/802/commits/a5978340d587361e6ffded32f8f5715740fbc20b